hudi-cli/src/main/java/org/apache/hudi/cli/commands/ClusteringCommand.java

```diff
@@ -59,7 +58,7 @@ public String scheduleClustering(
     SparkLauncher sparkLauncher = SparkUtil.initLauncher(sparkPropertiesPath);
 
     // First get a clustering instant time and pass it to spark launcher for scheduling clustering
-    String clusteringInstantTime = HoodieActiveTimeline.createNewInstantTime();
+    String clusteringInstantTime = client.getActiveTimeline().createNewInstantTime();
```

Comment:
Kind of think we should avoid to instantiate the active timeline where possible, the active timeline construction actually triggers the metadata files listing, which is not relevent with time generation.

Reply:
Moved to metaClient